fuzzing/private/oss_fuzz/repository.bzl

```diff
@@ -14,19 +14,51 @@
 
 """Repository rule for configuring the OSS-Fuzz engine and instrumentation."""
 
+_UBSAN_STANDALONE_CXX_LIB_NAME = "libclang_rt.ubsan_standalone_cxx-x86_64.a"
+
 def _to_list_repr(elements):
     return ", ".join([repr(element) for element in elements])
 
+def _find_llvm_lib(repository_ctx, target_file):
+    result = repository_ctx.execute([
+        repository_ctx.which("bash"),
+        "-c",
+        """
+        set -euf -o pipefail
+        set -x
+        find "$({llvm_config} --libdir)" -name {target_file} | head -1
+        """.format(
+            llvm_config = "llvm-config",
+            target_file = target_file,
+        ),
+    ], quiet = False)
+    file_path = result.stdout.strip()
+
+    if result.return_code != 0 or not file_path:
+        fail("Could not find LLVM library '%s'" % target_file)
+    return file_path
+
 def _extract_build_params(
         repository_ctx,
         fuzzing_engine_library,
+        sanitizer,
         cflags,
         cxxflags):
     stub_srcs = []
     stub_linkopts = []
     instrum_conlyopts = []
     instrum_cxxopts = []
 
+    if sanitizer == "undefined":
```

Comment:
Do we need to check arch is x64 or should we support both via sustituting the arch in filename (can skip for now, can add todo for later too if needed).

Reply:
That's a good point, it's actually easy to parameterize this by the architecture. I've just done so.